cms/release_calendar/blocks.py

```diff
@@ -13,7 +18,7 @@
 class ContentSectionBlock(blocks.StructBlock):
     """A content section with list of links."""
 
-    title = blocks.CharBlock()
+    title = blocks.CharBlock(label=_("Section title"))
```

Comment:
```suggestion
    title = blocks.CharBlock(label="Section title")
```

this is admin only, so doesn't need to be translatable